### PR TITLE
Update Ubuntu Advantage Service Description

### LINKED ISSUE #2282: Build UA service description update

### PR DESCRIPTION
**Pull request by deadlight**
## Done

Update Ubuntu Advantage Service Description to the latest version

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/legal/ubuntu-advantage/service-description](http://0.0.0.0:8001/legal/ubuntu-advantage/service-description)
- Check against list of highlighted changes in the [copydoc](https://docs.google.com/document/d/1mrNrsC1WDf6QTg3nrVCEZS8vJzGz1fokvoFDVC4s_mI/edit#)

## Issue / Card

Fixes #2282

### CODE REVIEW

**Comment by cjohnston1158** [comment]
TOC still says "The Canonical Distribution of Kubernetes ›"

**Comment by cjohnston1158** [comment]
This isn't a new problem, but 2.3.x, 2.4.x, and 2.5.x have bullets that look really bad because they are to the left of everything else.

**Comment by cjohnston1158** [comment]
- 3.2“Business hours” <--- there should be a space between 2 and "
- location is agreed All times  <--- there should be a period between agreed and All
- Table of response times <-- the alignment looks a little odd to me, with the head wrapping but nothing else wrapping I think.. I also wonder if there should be a separator between the columns, especially the standard/advanced column and the times column. Thoughts?
- Summary of service offerings <-- EssentialStandardAdvanced seem to run together
- Summary of service offerings <-- for FIPS in the table should we link to the relevant section like is done for the other rows?
- Certified Public Cloud partners can be found in the Ubuntu partner listing <-- does it make sense to have this somehow nested under the paragraph above it as it's referring to the CPC reference in that paragraph?
- TOC <--- missing FIPS

**Comment by deadlight** [comment]
@cjohnston1158 : 
Re: the severity-level table and general widths:
The Ubuntu design guidelines don't allow for text wider than we already have it (wider blocks are hard for readers to parse, especially for people with sight/reading impairments). I think that the best way to deal with the incongruously-wide table is to switch it to normal, inline content rather than a table. This works well because of the length of the severity-level descriptions.

**Comment by deadlight** [comment]
@cjohnston1158 I think that I've addressed everything on your list:  http://www.ubuntu.com-pr-2376.run.demo.haus/legal/ubuntu-advantage/service-description

It looks like I'll have to wait for Monday morning for sign-offs from others so I'll update the date to then.

**Comment by cjohnston1158** [comment]
- I think FIPS in the Summary of service offerings should point to Advanced like KVM and and LXD (to Standard) do. Thoughts?

**Comment by deadlight** [comment]
@cjohnston1158 That makes sense. I'll modify it.

File: templates/legal/ubuntu-advantage/service-description.html [hunk 1]
```diff
@@ -223,413 +197,453 @@ <h2 id="assurance">5. Assurance</h2>
...
-</thead>
-<tbody>
-<tr>
-<td>24x7 self-service customer care portal and knowledge base (see <a href="#ua-support-scope">Section 2</a>)</td>
-<td class="u-align--center"><img width="16" height="16" alt="Yes" src="{{ ASSET_SERVER_URL }}c4b02d61-tick-orange.svg" /></td>
-<td class="u-align--center"><img width="16" height="16" alt="Yes" src="{{ ASSET_SERVER_URL }}c4b02d61-tick-orange.svg" /></td>
-<td class="u-align--center"><img width="16" height="16" alt="Yes" src="{{ ASSET_SERVER_URL }}c4b02d61-tick-orange.svg" /></td>
-</tr>
-<tr>
-<tr>
```
Comment (grahambancroft) [inline templates/legal/ubuntu-advantage/service-description.html]: extra `<tr>`

**Pull request merged**